README.md

```diff
@@ -1,20 +1,18 @@
-ember-data
+EmberData
 ==============================================================================
 
 [![Build Status](https://github.com/emberjs/data/workflows/CI/badge.svg)](https://github.com/emberjs/data/actions?workflow=CI)
 [![Code Climate](https://codeclimate.com/github/emberjs/data/badges/gpa.svg)](https://codeclimate.com/github/emberjs/data)
 [![Discord Community Server](https://img.shields.io/discord/480462759797063690.svg?logo=discord)](https://discord.gg/zT3asNS)
 
-`ember-data` is a library for robustly managing data in applications built with
-[Ember.js](https://github.com/emberjs/ember.js/).
 
-`ember-data` is designed to be agnostic to the underlying persistence
-mechanism, so it works just as well with `JSON API` over `HTTP` as it does
-with streaming `WebSockets` or local `IndexedDB` storage.
+# Overview
 
-It provides many of the facilities you'd find in server-side `ORM`s like
-`ActiveRecord`, but is designed specifically for the unique environment of
-`JavaScript` in the browser.
+`EmberData` is a lightweight reactive data library for javascript applications that provides composable primitives for ordering query/mutation/peek flows, managing network and cache, and reducing data for presentation that you can plug-and-play as desired for any api  structure and format.
```

Comment:
```suggestion
`EmberData` is a lightweight reactive data library for JavaScript applications that provides composable primitives for ordering query/mutation/peek flows, managing network and cache, and reducing data for presentation. You can plug-and-play as desired for any api  structure and format.
```

It would be good so split this into 2 sentences.

Is there a catchy, jargon free (or light jargon) phrase you can open with?